### PR TITLE
Fixes to non-active accounts and disabled buttons

### PR DESCRIPTION
**Pull request by benzap**
- Fixes Issue #313
  - for submit, vote and challenge pages

- Fixes Issue #314
- Fixes Issue #315 

# Previews

# 313
a

# 314
b

# 315
c

### CODE REVIEW

**Comment by madvas** [comment]
314 - Does it display "You don't have enough DANK to challenge this meme" for users that don't have enough DANK, as I described in the issue? 
315 - Are those inputs disabled as well? If not, please make it so. On Meme Detail page as well

**Comment by benzap** [comment]
@madvas 
for 314, i'll have to include that
for 315, missed this as well, i'll throw down a commit with the changes

**Comment by benzap** [comment]
@madvas, pushed a new commit with the other changes

    More fixes
    - meme detail vote panel fixes
      - disabled input text when no active account
      - displays red letter text to user regarding lack of DANK coin if account balance is zero
    - meme detail challenge panel fixes
      - displays red letter text to user regarding lack of DANK coin if account balance is zero
    - vote page
      - disabled input text when no active account
      - displays red letter text to user regarding lack of DANK coin if account balance is zero
    - challenge page
      - displays red letter text to user regarding lack of DANK coin if account balance is zero

File: src/memefactory/ui/meme_detail/page.cljs [hunk 1]
```diff
@@ -472,8 +480,11 @@
           (if @tx-success?
             "Voted"
             "Vote Stank")]]]
-       [:div "You can vote with up to " (format/format-token (quot @balance-dank 1e18) {:token "DANK"})]
-       [:div "Token will be returned to you after revealing your vote."]])))
+       (if (> @balance-dank 0)
```
Comment (madvas) [inline src/memefactory/ui/meme_detail/page.cljs]: Isn't @balance-dank BigNumber here? In that case would need to use `bn/>` function

Comment (benzap) [reply src/memefactory/ui/meme_detail/page.cljs]: I don't believe it is BigNumber, tests show that it's working, and the original balance was showing correctly. There was some discussion a while back about how the values aren't stored in BigNumber format within SQLite, so I wonder if that's the reason.

Comment (benzap) [reply src/memefactory/ui/meme_detail/page.cljs]: Might make sense to make the changes anyways, i'll throw up another commit

Comment (madvas) [reply src/memefactory/ui/meme_detail/page.cljs]: yeah, I don't mean SQLite's BigNumber, but bignumber that comes from web3.js, from [district-ui-account-balances](https://github.com/district0x/district-ui-web3-account-balances) subscription

**Comment by benzap** [comment]
@madvas latest commit has bignumber equivalent operators, however I didn't replace what was already there. I think we should start another ticket for additional work

**Comment by madvas** [comment]
Okay thank you

**Pull request merged**